Documentation/Signoffs/Resistance_System_Control.md

```diff
@@ -0,0 +1,127 @@
+Team 4 – Mario Kart & Ride Replay Simulator
+
+Team Members: Ray Durlin, Blake Pickett, Tyler Chittum, Benjamin Reed, and Sage Mooneyham
+
+## Detail Design: Resistance System Control Circuit ##
+
+Please note: the scope of this signoff extends only to the control cicuit itself, not to housing components or connectors. Also, the signoff document for the resistance system contains BOM listings and information redundant to this document.
+
+### Function of the Subsystem:
+
+- This subsytem acts to control the linear actuator used in the ride resistance system.
+- Controls both the direction and speed of the linear actuator.
+- Takes an input from the ride replay subsystem over a wireless connection, which determines the distance set point for the linear actuator.
+- Takes an input from the linear actuator feedback to determine the current position of the actuator.
+
+### Constraints:
+
+#### Actuator Duty Cycle
+
+The duty cycle of the utilized linear actuator is 25 %. This value must not be exeeded during normal operation.
+
+#### Actuator Driving Voltage
+
+The utilized actuator is rated for operation at 12 VDC. The control circuit must be capable of providing this volatage to the actuator.
+
+#### Current Capability
+
+The utilized actuator draws a stall current of 7 A, and has a nominal current draw from 1.1 A to 3.4 A. The control circuit must be capable of providing this current to the linear actuator without undergoing thermal overload. In addition, the utilized power connector must be capable of handling the total current demand.
+
+#### Wireless Communications
+
+The control ciruit must be capable of recieving a wirelessly-transmitted digital signal using Bluetooth.
+
+#### Back-EMF Protection
+
+In the course of operation, particularly when the motor comes to a halt after running, a transient voltage potential known as back-EMF will be generated by the motor with opposite polarity to its driving voltage. Due to this, the control cicuit must not be vulnerable to damage from transient reverse-polarization.
+
+#### Mountability
+
+The PCB layout must account for the need to mount the circuit within its housing.
+
+#### Stability
+
+The device must operate without becoming unstable.
+
+#### Resolution
+
+The device must be capable of providing at least 85 unique states, or lengths of extension.
+
+### Schematic and PCB Layout
```

Comment:
Is there are reason to not use an off the shelf H bridge? You are welcome to implement your own if you are explicitly wanting to do so. But, that introduces an unnecessary step.

Reply:
I like the idea of fitting everything on a PCB, as opposed to running pin socket cables between several pre-made modules as I think it is a more streamlined solution which simplifies mounting (you only have to attach one board, rather than three). It also allows for the use of a specific power connector type (AMASS XT30) which will allow easy connection and disconnection of this subsystem to the power subsystem and prevent the user from connecting the power connection in reverse.